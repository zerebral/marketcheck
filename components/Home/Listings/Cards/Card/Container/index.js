import styled from 'styled-components'
import { em } from 'polished'
import { percentage, mediaMax } from '%/styles/mixins'

const CardContainer = styled.div`
  padding: 0 ${em(9)};
  max-width: ${percentage(1, 3)};
  flex-basis: ${percentage(1, 3)};
  ${mediaMax.laptop`
    max-width: 50%;
    flex-basis: 50%;
    margin-bottom: ${em(30)};
  `}
  ${mediaMax.tablet`
    max-width: 100%;
    flex-basis: 100%;
  `}
`
export default CardContainer
