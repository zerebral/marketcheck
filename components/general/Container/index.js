import styled from 'styled-components'

export default styled.div`
  max-width: 1170px;
  margin-right: auto;
  margin-left: auto;
`
