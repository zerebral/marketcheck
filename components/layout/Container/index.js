import { clearFix, rem } from 'polished'
import styled from 'styled-components'
import { grid } from '%/styles'

const padding = rem(grid.gutter / 2)

const Container = styled.div`
  ${clearFix()}
  position: relative;
  margin: 0 auto;
  max-width: ${rem(1170 + grid.gutter)};
  padding: 0 ${padding};
`

export default Container
