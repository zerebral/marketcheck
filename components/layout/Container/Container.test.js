/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import Container from './index'

describe('Container Component', () => {
  test('Container Passes the snapshot', () => {
    const component = renderer.create(<Container />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
