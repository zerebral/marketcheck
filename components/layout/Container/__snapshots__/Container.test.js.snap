// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Component Container Passes the snapshot 1`] = `
<div
  className="Container-s1gwg6so-0 jOdNxa"
/>
`;
